jira_analysis/cycle_time/chart/ticket_control.py

```
@@ -16,6 +16,8 @@ def generate_control_chart(
     tickets: List[Issue], chart_class: Type[IChart] = Chart
 ) -> None:
     if not tickets:
+        print('No data. Check that the JSON file was correctly populated ' +
```

I want to keep `print` statements out of this part of the code.

If we go this route, it's better to raise an exception for higher-level code to catch and report.